### PR TITLE
Fix

### PR DESCRIPTION
**Pull request by krasa**

### CODE REVIEW

File: src/main/java/ch/frankel/blog/reflection/MutablePerson.java [hunk 1]
```diff
@@ -4,9 +4,9 @@
 
 public class MutablePerson implements Person {
 
-    private String firstName;
-    private String lastName;
-    private Date birthDate;
+	String firstName;
```
Comment (nfrankel) [inline src/main/java/ch/frankel/blog/reflection/MutablePerson.java]: Why open the visibility?

Comment (krasa) [inline src/main/java/ch/frankel/blog/reflection/MutablePerson.java]: It was needed for ReflectASM FieldAcces

**Comment by nfrankel** [comment]
A great PR, thanks for it!
Just would you mind following Sun's Java conventions:

* methods should start with a lower case character
* no underscore, but camel case

**Comment by krasa** [comment]
It is more readable without camel case.

**Comment by nfrankel** [comment]
That's a very personal point of view. I do agree with it when the method name is very long, as in tests. Still, it's Java code and camel case is the convention.

**Comment by krasa** [comment]
JMH samples itself do not stick with CamelCase, as well as other people http://stackoverflow.com/a/22337726/685796
Underscores are simply needed to see a type of the test without reading the whole name.

**Pull request merged**